### DISCUSSION TITLE
Welcome to pharm_sales Discussions!

### DISCUSSION THREAD

**Discussion by tutorialcreation** [tutorialcreation/pharm_sales#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tutorialcreation** [comment]
ok so who wants to talk to me, github is really cool by the way

**Reply by rafaesam** [reply]
I just saw this, it seems like cool to have a discussion here.. So my question on how to start the time series analysis?
Another one is on the dvc, I'm not sure what is the best way to setup the env. every time I'm trying to add new file it tells me that command is not specified I mean (dvc)
